packages/story/src/lib/Notion.tsx

```diff
@@ -1,4 +1,4 @@
-import { Notion, type Block } from "@notionpresso/react";
```

Comment:
p1: don't import with relative path

it is mono repo

see: workspace property on root package.json

Reply:
I have resolved this issue.